open-banking-accelerator/accelerators/ob-apim/repository/resources/apis/DynamicClientRegistration/dcr-swagger.yaml

```diff
@@ -0,0 +1,370 @@
+securityDefinitions:
+  TPPOAuth2Security:
+    type: oauth2
+    flow: application
+    scopes:
+        cdr-registration: "Ability to call registration endpoint"
+    x-scopes-bindings:
+      cdr-registration: "Internal/subscriber"
+    description: >-
+      TPP client credential authorisation flow with the ASPSP. No scopes defined
+      as per specification
+    tokenUrl: 'https://authserver.example/token'
+definitions:
+  RegistrationProperties:
+    type: object
+    required:
+      - client_id
+      - client_name
+      - client_description
+      - client_uri
+      - org_id
+      - org_name
+      - redirect_uris
+      - logo_uri
+      - jwks_uri
+      - revocation_uri
+      - software_id
+      - software_roles
+      - scope
+      - token_endpoint_auth_signing_alg
+      - token_endpoint_auth_method
+      - grant_types
+      - response_types
+      - software_statement
+    properties:
+      client_id:
+        description: Data Holder issued client identifier string
+        type: string
+        example: '35a5a70b-5b8d-41f4-9cbd-96cfbc15c58a'
+      client_id_issued_at:
+        description: >-
+          Time at which the client identifier was issued expressed as seconds
+          since 1970-01-01T00:00:00Z as measured in UTC
+        type: integer
+        format: int32
+        example: 1571808167
+      client_name:
+        description: 'Human-readable string name of the software product to be presented to the end-user during authorization'
+        type: string
+        example: 'Mock Software'
+      client_description:
+        description: 'Human-readable string name of the software product description to be presented to the end user during authorization'
+        type: string
+        example: 'A mock software product'
+      client_uri:
+        description: 'URL string of a web page providing information about the client'
+        type: string
+        example: 'https://www.mockcompany.com.au'
+      org_id:
+        description: 'A unique identifier string assigned by the CDR Register that identifies the Accredited Data Recipient Brand'
+        type: string
+        example: '3B0B0A7B-3E7B-4A2C-9497-E357A71D07C8'
+      org_name:
+        description: 'Human-readable string name of the Accredited Data Recipient to be presented to the end user during authorization'
+        example: 'Mock Company Inc.'
+        type: string
+      redirect_uris:
+        description: 'Array of redirection URI strings for use in redirect-based flows'
+        type: array
+        items:
+          type: string
+          format: uri
+        example: ['https://www.mockcompany.com.au/redirects/redirect1', 'https://www.mockcompany.com.au/redirects/redirect2']
+      logo_uri:
+        description: 'URL string that references a logo for the client. If present, the server SHOULD display this image to the end-user during approval'
+        type: string
+        example: 'https://www.mockcompany.com.au/logos/logo1.png'
```

Comment:
these urls r reffering au side links.Can we move out them too..

Reply:
Fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/c257729b9462679e6c0b023eb3cefbab7c45167c